src/parser/jobs/gnb/modules/OGCDDowntime.tsx

```diff
@@ -11,6 +11,10 @@ const FIRST_USE_OFFSET_NO_MERCY = 10000
  */
 const FIRST_USE_OFFSET_BLOODFEST = 22000
 
+const FIRST_USE_OFFSET_BOWBLAST = 17500 // The BowBlast combo is appiled at a maximum of 7 GCDs in current viable openers but can be applied as soon as 5 GCDs
```

Comment:
Ultranit, to reduce further maintenance if this ever changes, would you not want to separate Bowshock and Pewpew Zone? Also did you validate on high damage JP GNBs that they're not doing some weird opener? Obviously don't need to look at low end weirdness, but might be good for high end. NBD if you did tho.

Reply:
Updated the files to the 4 current openers, and separated the offsets to allow for easier changes should a opener fall out of favor and the skills are used sooner.

(Also pewpew zone is here to stay now)

Reply:
![GNB_openers](https://user-images.githubusercontent.com/37195907/78418668-6a1a1200-760c-11ea-99f6-a09ecb478baf.jpg)